frontend/javascripts/admin/voxelytics/workflow_list_view.tsx

```diff
@@ -84,9 +89,18 @@ export default function WorkflowListView() {
     }
   }
 
-  usePolling(loadData, VX_POLLING_INTERVAL);
+  usePolling(async () => {
```

Comment:
@fm3 I tried to address your comment about requesting the workflows twice while trying not to mess around in the code too much. I thought I could delete the `loadData` call where the search query is persisted, but then a dataless page is rendered. So I think this might be an acceptable solution for now, but I am keen to hear what you say :)